features/step_definitions/edit_comment_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^a comment has been made$/) do
+  @otherComment = FactoryGirl.create(:comment,
```

Comment:
Style/VariableName: Use snake_case for variable names. (https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)